### DISCUSSION TITLE
Fever API

### DISCUSSION THREAD

**Discussion by thagoat** [nkanaev/yarr#225] [Q&A] [answered]
Has anyone gotten this to work? I get a 404 when navigating to the /fever endpoint, and all the apps I've tried error out either 404, or malformed .json

Jusr curious, I'm totally digging this project otherwise, and maybe I can cut my dependence on ToxicToxic-RSS

**Comment by Shinokuni** [comment]
I second this. I am the developer of [Readrops](https://github.com/readrops/Readrops), an Android RSS client with the Fever API implemented. I currently can't test yarr Fever implementation as `/fever/` endpoint returns me a 404 error.

**Reply by thagoat** [reply]
I've tested v2.5 with a few apps, with mixed results in the sync function. But your app sends an auth error every time, and I haven't had that error with any other app. Just a heads up, not looking for support 😄

**Comment by arsfeld** [comment]
I’m using it on Reeder Classic in iOS and it works fine.

The only issue I have is that it’s not in any released version, so I’m using a random commit number with the fever API included.

**Reply by valpackett** [reply]
@nkanaev can we get a new release please?

**Comment by nkanaev** [comment, marked as answer]
Please check out v2.5 to see if it's working.

**Comment by thagoat** [comment]
Working! Thank you for all the hard work. And all the contributors!
